package.json

```diff
@@ -65,9 +65,22 @@
     "rimraf": "^3.0.0",
     "sha.js": "^2.4.11",
     "simple-git": "^1.118.0",
-    "touch": "^3.1.0"
+    "touch": "^3.1.0",
+    "tsc": "^1.20150623.0",
+    "highlight.js": "^9.18.1",
+    "vue": "^2.5.22",
+    "vue-property-decorator": "^7.0.0",
+    "vue-router": "^3.1.6",
+    "vuex": "^3.0.1",
+    "vuex-module-decorators": "^0.10.1"
   },
   "devDependencies": {
+    "@storybook/addon-links": "^5.3.14",
+    "@storybook/addon-options": "^5.3.14",
+    "@storybook/preset-scss": "^1.0.2",
+    "@storybook/preset-typescript": "^2.1.0",
```

Comment:
Why are these packages added to the actual package.json for developing rdvue? That is to say, does compiling the tool depend on these packages? If not, they should be removed.

Reply:
They weren't supposed to be there and has been removed.